src/main/java/org/alfresco/rest/api/impl/QuickShareLinksImpl.java

```diff
@@ -618,6 +656,21 @@ private QuickShareLink getQuickShareInfo(NodeRef nodeRef, Map<String, Object> ma
         }
     }
 
+    private Node getNode(Node node, NodeRef nodeRef, List<String> includeParam)
```

Comment:
Or remove this completely and just do it at the start of the calling method ... otherwise, I think it ends up calling nodeService.getProperties twice ?

Reply:
There is no clean way of doing this, nodeService.getProperties is actually called 3 times (3rd time to get the map) and this was like that before my changes. I don't see an easy way to solve this without a big refactoring.